src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/SecretsMasker.java

```diff
@@ -84,7 +84,13 @@ protected Class<TaskListenerDecorator> type() {
 
         @Override
         protected TaskListenerDecorator get(DelegatedContext context) throws IOException, InterruptedException {
-            KubernetesComputer c = context.get(KubernetesComputer.class);
+            KubernetesComputer c;
+            try {
+                c = context.get(KubernetesComputer.class);
+            } catch (IOException | InterruptedException x) {
+                LOGGER.log(Level.FINE, "Unable to look up KubernetesComputer", x);
+                return null;
```

Comment:
Minor production code change. IIRC this was necessary with an earlier iteration of https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/180 but not after a redesign in https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/180/commits/02fbe3afa47d9e317fe826a2c24ad792bb316703. At any rate, this seems like a prudent robustness change: if we cannot even determine whether there is a `KubernetesComputer` associated with some step, we should err on the side of assuming that there is not, and that we need not bother masking anything, rather than failing to do anything at all.